scripts/translation-status.ts

```diff
@@ -332,6 +358,21 @@ class TranslationStatusBuilder {
 		return lines.join('\n');
 	}
 
+	renderTranslationNeedsReview(prs: PullRequest[]) {
+		const lines: string[] = [];
+
+		if (prs.length > 0) {
+			lines.push(`<ul>`);
+			lines.push(
+				...prs.map((pr) => `<li>` + `${this.renderLink(pr.html_url, pr.title)} ` + `</li>`)
```

Comment:
What do you think of stripping backticks out of the PR titles?

```suggestion
				...prs.map((pr) => {
					const title = pr.title.replaceAll('`', '');
					return `<li>` + this.renderLink(pr.html_url, title) + `</li>`;
				})
```

There are a few in your screenshot and it’s a bit of a nit pick, but I guess we can remove them? 
![PR titles rendered including backticks](https://user-images.githubusercontent.com/357379/227244853-66e68eb2-c38e-4b74-9207-2649c0de4e35.png)

Reply:
Nice. yeah, that'll look much better